### PR TITLE
Bay merges

### PR DESCRIPTION
**Pull request by silveryferret**
On the tin. Tested on local, every file touched still seems to be working as far as I can tell, feel free to look over my work. 

Also snags https://github.com/Baystation12/Baystation12/pull/14774 for future use.

### CODE REVIEW

File: code/modules/emotes/definitions/human.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /decl/emote/human
 	key = "vomit"
 
-/decl/emote/human/check_user(var/atom/user)
-	return istype(user, /mob/living/carbon/human)
+/decl/emote/human/check_user(var/mob/living/carbon/human/user)
+	return (istype(user) && user.check_has_mouth() && !user.isSynthetic())
```
Comment (ShadowLarkens) [inline code/modules/emotes/definitions/human.dm:5]: I believe this is going to make `*deathgasp` fail for synthetics. It'll be inherited behaviour.

File: code/modules/eros/clothing/suits.dm [hunk 2]
```diff
@@ -133,13 +102,12 @@
 /obj/item/clothing/suit/storage/hooded/wintercoat
 	name = "winter coat"
 	desc = "A heavy jacket made from 'synthetic' animal furs."
-	icon_state = "eros_coatwinter"
-	item_state_slots = list(slot_r_hand_str = "eros_coatwinter", slot_l_hand_str = "eros_coatwinter")
+	icon_state = "coatwinter"
```
Comment (ShadowLarkens) [inline code/modules/eros/clothing/suits.dm:105]: We will probably have to revert all of these icons.

Comment (silveryferret) [reply code/modules/eros/clothing/suits.dm:105]: Can I ask why? Bay ported winter coats as well as ponchos, so we're using mostly their code as far as that goes now.

**Comment by silveryferret** [comment]
I'll leave this up for anyone else to look over, but I feel pretty confident about this merge. :ok_hand:

File: code/modules/eros/clothing/suits.dm (hunk 2 above)
Comment (ShadowLarkens) [reply code/modules/eros/clothing/suits.dm:105]: Do we not have any special sprites for them?

**Comment by OrbisAnima** [comment]
Excelent work.

**Pull request merged**